docs/user-manual/en/amqp-server-connections.md

```diff
@@ -0,0 +1,230 @@
+           <queue name="localQueueB"/>
+        </anycast>
+     </address>
+</addresses>
+
+```
+*Important:* Notice that on the `receiver` case we will only match local existent queues, so if you are using receivers make sure you pre-create the queue locally otherwise we broker will not be able to match remote queues and addresses.
+
+*Important:* Be careful to not create a sender and a receiver to the same destination, otherwise they will incur in infinite echoes of sends and receives.
+
+
+# Peers
+A peer broker connection element will be a mix of sender and receivers. The ActiveMQ Artemis broker will create both a sender and a receiver for a peer element, and the endpoint should know how to deal with the pair without creating an infinite feedback of sending and receiving messages.
+
+Currently [Apache QPID Dispatch Router](https://qpid.apache.org/components/dispatch-router/index.html) as a peer. ActiveMQ Artemis will create the pair of receivers and sender for each matching destination, and these senders and receivers will have special configuration to let qpid dispatch know to collaborate with an ActiveMQ Artemis Broker.
+
+You would be able to play with advanced networking scenarios with Apache QPID Dispatch Router and getting a lot of benefit out of the AMQP protocol and its eco system.
+
+With a peer, you have the same properties you would have on a sender and receiver. So the configuration is fairly similar to a sender or a receiver:
+
+```xml
+<broker-connections>
+    <amqp-connection uri="tcp://MY_HOST:MY_PORT" name="my-broker">
+       <peer match="queues.#"/>
+    </amqp-connection>
+</broker-connections>
+
+<addresses>
+     <address name="queues.A">
+        <anycast>
+           <queue name="localQueueA"/>
+        </anycast>
+     </address>
+     <address name="queues.B">
+     <anycast>
+        <queue name="localQueueB"/>
+     </anycast>
+    </address>
+</addresses>
+```
+
+*Important:* Do not use this feature to connect to another broker, otherwise any message send will be immediately ready to consume creating an infinite echo of sends and receives.
+
+# Mirror 
+The mirror option on the broker connection, can capture events from the broker and pass them over the wire to another broker, creating the possibility of haivng multiple asynchronous replicas.
+The events captured are the following:
+
+- Message routing
+- Message acknowledgement
+- Queue and Address Creation
+- Queue and Address Deletion
+
+When you configure mirror, these events are captured from the broker, stored on a local queue and later forward to a target destination on another ActiveMQ Artemis.
+
+The local queue is called source-mirror-address, and the target destination will be called target-mirror-address.
+
+
+*TODO: Picture showing events captuerd into sorce-mirror-address, and being transferred over AMQP towards target-mirror-address.
+
+
+All you have to do is to define the mirror element, where you have these optional arguments:
```

Comment:
Need details around bits like exclusive consumers, message groups and other queue mechnaics if a clients connect to both brokers would be broken.